packages/trpc-playground/src/handlers/common.ts

```diff
@@ -18,12 +18,12 @@ export const getCommonHandlerReqData = async (config: TrpcPlaygroundConfig) => {
     })
   }
 
-  const types = await resolvedConfig.resolveTypes(config.router)
+  const types = await resolvedConfig.resolveTypes!(config.router)
```

Comment:
`resolveTypes` can't be undefined, why is this here?

Reply:
I don't know why I did that, removed it.